docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -0,0 +1,229 @@
+
+
+|===
+|Helper |Use case |Examples
+
+|json
+|Serialize variables in JSON format
+|`{{json event}}` +
+`{{json event.key event.value}}` +
+`{{json filters=context.panel.filters}}`
+
+
+|rison
+|Serialize context variables in https://github.com/w33ble/rison-node[rison] format. Rison is a common format for {kib} apps for storing state in the URL.
+|`{{rison event}}` +
+`{{rison event.key event.value}}` +
+`{{rison filters=context.panel.filters}}` +
+
+
+|date
+|Format dates. Supports relative dates expressions (for example,  "now-15d"). Refer to the https://momentjs.com/docs/#/displaying/format/[moment] docs for different formatting options.
+|`{{ date event.from “YYYY MM DD”}}` + 
+`{{date “now-15”}}`
+|===
+
+
+[float]
+[[variables]]
+==== URL template variables
+
+The URL drilldown template has three sources for variables:
+
+. *Global* static variables that don’t change depending on the  place where URL drilldown is used or which user interaction executed the drilldown. For example: `{{kibanaUrl}}`.
+. *Context* variables that change depending on where the drilldown is created and used. These variables are extracted from a context of a panel on a dashboard. Example: ``{{context.panel.filters}}`` which gives access to filters that applied to the current panel.
+. *Event* variables that depend on the trigger context. These variables are dynamically extracted from the interaction context when the drilldown is executed.
+
+[[values-in-preview]]
+A subtle but important difference between *context* and *event* variables is that *context* variables use real values in previews when creating a URL drilldown. 
+For example, `{{context.panel.filters}}` are previewed with the current filters that applied to a panel. 
+*Event* variables are extracted during drilldown execution from a user interaction with a panel (for example, from a pie slice that the user clicked on).
+
+Because there is no user interaction with a panel in preview, there is no interaction context to use in a preview.
+To work around this, {kib} provides a sample interaction that relies on a picked <<trigger-picker, trigger>>.
+So in a preview, you might notice that `{{event.value}}` is replaced with `{{event.value}}` instead of with a sample from your data.
+Such previews can help you make sure that the structure of your URL template is valid.
+However, to ensure that the configured URL drilldown works as expected with your data, you have to retest the drilldown through the panel after saving a dashboard.
+
+You can access the full list of variables available for the current panel and selected trigger by clicking *Add variable* in the top-right corner of a URL template input.
+
+[float]
+[[variables-reference]]
+==== Variables reference
+
+
+|===
+|Source |Variable |Description
+
+|*Global*
+|
+| Static variables that don’t change depending on the place where a URL drilldown is used or which user interaction executed the drilldown.
```

Comment:
```suggestion
```

This row seems to be repeat from above bullet list:

![image](https://user-images.githubusercontent.com/9773803/92453435-8b84b880-f1bf-11ea-9124-a2dfcbf4391e.png)

Reply:
Right. I plan that `Help` link from `add variables` popover would link directly to this reference. So left this description in the table for users how navigated directly to it